README.md

```diff
@@ -1,26 +1,39 @@
 # AppCenter Reviews
 
-This repository is used for deploying applications to the [elementary flatpak
-repository][1]. You can view your review status by [looking at the open PRs][2].
+This repository is used for deploying apps to the [elementary AppCenter Flatpak
+repository][1]. You can view your or any other app's review status by checking
+[open pull requests][2].
 
-To submit an application, please use the [AppCenter dashboard][3], as it will
-speed up the review process and ensure all the needed data is entered correctly.
-If you want to publish a git repository that is not on GitHub, make sure you
-adhere to the [publishing requirements][4], then you can make a PR here.
+>⚠️ **Note:** The new AppCenter Dashboard is unfinished and under rapid
+development. If you have issues using it, please [report them][3].
+
+To submit an app, please use [AppCenter Dashboard][4], as it will speed up the
+review process and ensure all the needed data is entered correctly. If you want
+to publish an app that is not hosted on GitHub, your has an RDNN that does not
+match where the code is hosted, or are having an issue submitting with AppCenter
+Dashboard: first make sure your app adheres to the [publishing requirements][5],
+then submit a pull request against this repository following the structure
+below.
 
 ## Review Process
 
-All applications currently in the review queue can be viewed in the
-[pending PRs][2]. Once the PR is approved and merged by one of our reviewers, it
-will be published to our [flatpak repository][1]. If any questions or changes
-come up during the review process, our reviewer will tag the application author
-in the conversation (if possible).
+All apps and updates in the review queue are listed as [open pull requests][2].
+Once a pull request is approved and merged by a reviewer, that app will be
+published to the [AppCenter Flatpak repository][1]. If any questions or
+requested changes come up during the review process, the reviewer will tag the
+app developer in the conversation on the pull request if possible.
+
+If your app was added with AppCenter Dashboard, future releases and tagged
+commits will be automatically submitted for review and approval as well. If you
+opened a pull request against this repository, you will need to open a new pull
+request for each release, or use the AppCenter Dashboard to set up automatic
+submission of releases.
```

Comment:
This is not true. Even if you open a PR to this repo, we will be using this repo (or the flatpak repo) as the source of truth for the cron job.

Reply:
Oh! Gotcha. Neat. Fixing.